alpakaConfig.cmake

```diff
@@ -1089,6 +1098,11 @@ IF((ALPAKA_ACC_GPU_CUDA_ENABLE OR ALPAKA_ACC_GPU_HIP_ENABLE) AND ALPAKA_CUDA_COM
     SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${_ALPAKA_COMPILE_OPTIONS_STRING}")
 ENDIF()
 
+IF(NOT MSVC)
+    # dependency for std::future
+    TARGET_LINK_LIBRARIES("alpaka" INTERFACE  Threads::Threads)
```

Comment:
CMake is saying:
```
  INTERFACE library can only be used with the INTERFACE keyword of
  target_link_libraries
Call Stack (most recent call first):

```
Since the target alpaka is an `INTERFACE`  so we need to use `INTERFACE` here.

Reply:
but there is one space to much after `INTERFACE`, this I will fix after but like to wait for other reviews and suggestions first.

Reply:
I see, thanks for explanations!